tools/ncpms-api-crawler/ncpms-crawler.py

```
@@ -0,0 +1,162 @@
+        except Exception as e:
+            print(f"API 요청 에러: {e}")
+            continue
+            
+        items = []
+        if text.startswith("{"):
+            data = json.loads(text)
+            items = data.get("service", {}).get("list", [])
+            if isinstance(items, dict): items = [items]
+        else:
+            try:
+                root = ET.fromstring(text)
```

Ruff (0.15.10)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)